src/Controllers/StandortController.cs

```diff
@@ -50,4 +62,34 @@ public StandortController(EwsContext context)
 
         return await standorte.AsNoTracking().ToListAsync().ConfigureAwait(false);
     }
+
+    /// <summary>
+    /// Asynchronously gets the <see cref="Standort"/> for the specified <paramref name="id"/>.
+    /// </summary>
+    /// <param name="id">The standort id.</param>
+    [HttpGet]
+    [Route("{id}")]
+    public async Task<IActionResult> GetByIdAsync(int id)
+    {
+        var standort = await Context.Standorte
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.HTektonik)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.HQualitaet).ThenInclude(b => b.Codetyp)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.HFormationFels)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.HFormationEndtiefe)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.Schichten).ThenInclude(s => s.Qualitaet)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.Schichten).ThenInclude(s => s.CodeSchicht)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.Schichten).ThenInclude(s => s.HQualitaet)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.Vorkomnisse).ThenInclude(v => v.HTyp)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Bohrprofile).ThenInclude(b => b.Vorkomnisse).ThenInclude(v => v.HQualitaet).ThenInclude(h => h.Codetyp)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Ablenkung).ThenInclude(a => a.Codetyp)
+            .Include(s => s.Bohrungen).ThenInclude(b => b.Qualitaet).ThenInclude(q => q.Codetyp).SingleOrDefaultAsync(s => s.Id == id).ConfigureAwait(false);
```

Comment:
REQ: Die Query mit all den Includes sollte ausgelagert werden, ansonsten laufen wir Gefahr, dass wir nicht dieselben Daten laden wie in `GetAsync()`.